addons/api/package.json

```diff
@@ -33,6 +33,8 @@
     "ember-cli-mirage": "^2.2.0",
     "ember-copy": "^2.0.1",
     "ember-data": "^3.20.0",
+    "ember-data-model-fragments": "5.0.0-beta.2",
```

Comment:
I think this is actually gone now. @calcaide is that correct?

Reply:
@cameronperera that is correct is gone and merged to main [in this PR](https://github.com/hashicorp/boundary-ui/pull/1507/files#diff-3e27e6039dc4ff0f8e02759ce553da2ec6691658e4e0f1244483f5d564c8ff44L35) (kudos to @ZedLi !!)

@lisbet-alvarez perhaps this branch needs to be updated from main to get latest changes?

Reply:
Removed.